packages/ui/package.json

```diff
@@ -98,6 +98,7 @@
   },
   "peerDependencies": {
     "react": "^16.14.0",
-    "react-dom": "^16.14.0"
+    "react-dom": "^16.14.0",
+    "react-datepicker": "^3.4.1"
```

Comment:
This way we will have it for the development because it is a dev dep, and we will have a single version of react-datepicker in production as it is a peer dep.

Reply:
@aigoncharov updated